src/Telemetry/telemetry-init.php

```diff
@@ -13,6 +13,11 @@
 
 require_once __DIR__ . '/class-telemetry-system.php';
 
+// Only run if WordPress is loaded.
+if ( ! function_exists( 'add_action' ) ) {
+	return;
+}
```

Comment:
Was there a particular reason or issue for adding this? If not, I'll be removing it (as well as the change in `lint.php`).

Reply:
I've proceeded with the changes, but we can revert if needed.